background/get-addon-settings.js

```diff
@@ -674,6 +674,20 @@ chrome.storage.sync.get([...ADDON_SETTINGS_KEYS, "addonsEnabled"], (storageItems
           delete settings.links;
           madeAnyChanges = madeChangesToAddon = true;
         }
+
+        if (addonId === "fullscreen" && settings.hideToolbar !== undefined) {
+          // Transition v1.36 to v1.37
+          if (!settings.hideToolbar) {
+            settings.toolbar = "show";
+          } else if (settings?.hoverToolbar) {
```

Comment:
Why is optional chaining used here? Isn't `settings` guaranteed to be an object?